### DISCUSSION TITLE
Purchase

### DISCUSSION THREAD

**Discussion by N1337XYZ** [Akascape/CTkDesigner-Support#10] [General]
If I purchase, are updates included, or do I just purchase the latest version?

**Comment by N1337XYZ** [comment]
TabView widget also possible?

**Reply by Akascape** [reply]
@N1337XYZ Working on tabview, you get access to all future version.

**Reply by N1337XYZ** [reply]
Great. Will wait for TabView then ;)

**Comment by teijiIshida** [comment]
Hello, 

I tested the trial. It looks great. But I really need tabview for my program. Is there any eta on tabview feature?

**Reply by Akascape** [reply]
@teijiIshida Currently not working on ctk tabview, but I will try to implement this. There are some issues with that widget so I have to modify it.
However there is pages + fixed  navigation buttons currently which work like tabs. 
 
But you can wait for the next versions. Maybe this month.
